.github/workflows/test-build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Testing Builds
+
+on:
+  push:
+  schedule:
+    - cron: '0 9 * * *' # every day at 9:00
```

Comment:
Since every push is checked, do we need to run the task on a daily basis?

Reply:
On daily basis, it runs only for the `main` branch. And as we are only building changes that are coming to `main` branch, I think that is also important that the `main` branch is still stable after all those changes.

I know that if PR is ok, also the `main` branch should be ok. I took this practice from the cookie-consent-manager.

Reply:
Removed.

Reply:
OK 🙂. I believe it would be redundant because there should be no way to get things into `main` other than through a PR.